presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveDynamicPartitionPruning.java

```diff
@@ -63,11 +63,11 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         return HiveQueryRunner.builder()
-                .setInitialTables(getTables())
-                .setHiveProperties(ImmutableMap.of("hive.dynamic-filtering-probe-blocking-timeout", "1h"))
                 // Reduced partitioned join limit for large DF to enable DF min/max collection with ENABLE_LARGE_DYNAMIC_FILTERS
                 .setSingleExtraProperty("dynamic-filtering.large-partitioned.max-distinct-values-per-driver", "100")
```

Comment:
slightly tangential to this PR, but this property seems to get overriden, since `setSingleExtraProperty` replaces the previously set properties?

Reply:
Good catch.

Reply:
Added a fix for this.

cc @raunaqmorarka